IPython/core/magics/execution.py

```diff
@@ -38,7 +38,7 @@
 from IPython.utils import py3compat
 from IPython.utils.py3compat import builtin_mod, iteritems, PY3
 from IPython.utils.contexts import preserve_keys
-from IPython.utils.io import capture_output
+from IPython.utils.capture import capture_output
```

Comment:
Yeah, I actually made the test fail while cleaning io, and rebase/squashed, but don't tell anyone.